cmd/validate.go

```diff
@@ -17,36 +17,36 @@ import (
 var ErrValidationErrorsFound = errors.New("errors found during validation")
 
 func validate(log *slog.Logger, cfg *config.Config, _ chan os.Signal) error {
-	if !cfg.PersistedOperations.Enabled {
-		err := fmt.Errorf("persisted operations must be enabled to run validate")
+	// We need a store defined to have files to validate
+	if cfg.PersistedOperations.Store == "" {
+		err := fmt.Errorf("store must be defined to have files to validate")
 		log.Error("Error running validations", "err", err)
 		return err
 	}
 
-	remoteLoader, err := persistedoperations.RemoteLoaderFromConfig(cfg.PersistedOperations, log)
-	if err != nil && !errors.Is(err, persistedoperations.ErrNoRemoteLoaderSpecified) {
-		log.Warn("Error initializing remote loader", "err", err)
-	}
-
-	po, err := persistedoperations.NewPersistedOperations(log, cfg.PersistedOperations, persistedoperations.NewLocalDirLoader(cfg.PersistedOperations, log), remoteLoader)
+	// Load the persisted operations from the local dir into memory
+	persistedOperations, err := persistedoperations.NewPersistedOperations(log, cfg.PersistedOperations, persistedoperations.NewLocalDirLoader(cfg.PersistedOperations, log), nil)
```

Comment:
will this not throw an NPE on the `nil` for the remoteLoader?

Reply:
nope theres a nil check built in somewhere deeper in the code